xds/src/main/java/io/grpc/xds/XdsClient.java

```diff
@@ -178,8 +179,7 @@ abstract static class CdsUpdate implements ResourceUpdate {
 
     abstract ClusterType clusterType();
 
-    // Endpoint-level load balancing policy.
-    abstract LbPolicy lbPolicy();
+    abstract ImmutableMap<String, ?> lbPolicyConfig();
```

Comment:
How would you feel about making this a PolicySelection instead?

Reply:
It would make sense. Gets us away from double parsing the config. It is just a question of how closely we want to adhere to the design vs. make optimizations.